### DISCUSSION TITLE
OAuth Provider Example Synology SSO

### DISCUSSION THREAD

**Discussion by xeo102** [mealie-recipes/mealie#4118] [Q&A]
Hi everyone, does anyone have a working configuration for Synology SSO? Unfortunately mine doesn't work:
```
OIDC_AUTH_ENABLED: true
OIDC_SIGNUP_ENABLED: true
OIDC_CONFIGURATION_URL: https://sso.example.synology.me/webman/sso/.well-known/openid-configuration
OIDC_CLIENT_ID: Client-ID-String
OIDC_USER_GROUP: users_mealie
OIDC_ADMIN_GROUP: administrators
OIDC_AUTO_REDIRECT: true
OIDC_PROVIDER_NAME: Synology
OIDC_REMEMBER_ME: true
```
Thanks in advance

**Comment by MichaelSala** [comment]
Hi,

You can try to add this :
OIDC_NAME_CLAIM: username

Some field needed bye mealie wasn't available with the mealie' default field name ... like name which is required for sign in/login !

Kind regards
Michael

**Comment by JoTec2002** [comment]
You can try following this guide:

https://github.com/mealie-recipes/mealie/discussions/4667
